packages/world/src/utils/InventoryUtils.sol

```diff
@@ -87,7 +87,7 @@ function removeAnyFromInventoryCount(EntityId playerEntityId, ObjectTypeId objec
       remaining -= spend;
     }
   }
-  require(remaining == 0, "Not enough objects");
+  require(remaining == 0, "Not enough objects in the inventory");
```

Comment:
Made this string match the other place we do a similar check

Reply:
do you think eventually we should move to custom errors? should also help a bit with contract size

Reply:
not needed for now though, can be changed later

Reply:
Seems like a good idea, lets create a separate issue to track it?